src/base/PolymeshTransactionBase.ts

```diff
@@ -383,13 +348,43 @@ export abstract class PolymeshTransactionBase<
     /* eslint-enable default-case */
   }
 
+  /**
+   * Return whether the transaction can be subsidized. If the result is false
+   *   AND the caller is being subsidized by a third party, the transaction can't be executed and trying
+   *   to do so will result in an error
```

Comment:
I thought the caller would still be able to make the transaction, but would have to cover the cost themselves.

Does this mean a subsidized account can do less things, even if they have the funds to pay for them?

Reply:
That's right. They can unilaterally exit the subsidizer relationship though if they need to run transactions that can't be subsidized

I proposed changes to core to address this in a cleaner way (https://polymathnetwork.slack.com/archives/C01RLA867C5/p1640281223354100?thread_ts=1640121955.343700&cid=C01RLA867C5), but there wasn't much interest

In any case, we're a long way from this becoming a real problem (if it ever does), and there's already a workaround

Reply:
Thanks, I wanted to be sure I was understanding this correctly.

Making a second key isn't too bad of a work around.